.travis.yml

```diff
@@ -28,7 +28,10 @@ jobs:
 
     - name: macOS | x86_64 | Python 3.x
       os: osx
-      env: PYTHON=python3
+      osx_image: xcode10.2
+      env:
+        - PYTHON=python3
+        - PYTEST_ADDOPTS='-k "unit_test or test_0_basic" --suppress-no-test-exit-code'
```

Comment:
why this?

Reply:
https://github.com/joerick/cibuildwheel/pull/430#issuecomment-702882820:

> Also, Travis CI's macOS job keeps on timing out. I guess we should limit the amount of tests, similar to AppVeyor?

To stay within 50 minutes. I think it's because we added 3.9 and didn't remove 3.5 (yet), that we are running too many builds now, for Travis CI's macOS builders to handle within 50 minutes. If you have other suggestions, please say so! :-)